css/main.css

```diff
@@ -169,3 +182,26 @@ label {
     height: 560px;
     background: #34495e;
 }
+
+.tooltip {

```

Comment:
As you noted, naming is inconsistent

Reply:
Yes I fixed this as well as the others I was talking about